### PR TITLE
starter code for webapp

### PR DESCRIPTION
**Pull request by JoyKiran**
Created and initialized angular app, removed all default html, css generated by angular team and kept simple as possible, so that new featured can be added in later stage.

### CODE REVIEW

File: tsconfig.app.json
Comment (JoyKiranSR) [inline tsconfig.app.json:1]: I don't think the commented line is necessary. Its better to remove it.

File: tsconfig.spec.json
Comment (JoyKiranSR) [inline tsconfig.spec.json:1]: I don't think the commented line is necessary here as well. Its better to remove it.

Comment (JoyKiran) [reply tsconfig.spec.json:1]: Oh yeah. Will do

File: tsconfig.app.json
Comment (JoyKiran) [reply tsconfig.app.json:1]: Yep. I will remove it. Thanks

**Review by JoyKiranSR — CHANGES_REQUESTED** [review]
Remove unnecessary comments

File: src/app/app.component.spec.ts
Comment (JoyKiranSR) [inline src/app/app.component.spec.ts:1]: Is the commented test case necessary? If not better to remove it.

Comment (JoyKiran) [reply src/app/app.component.spec.ts:1]: Yes. Its required for further unit testing. Just keeping it for reference per se.

Comment (JoyKiranSR) [reply src/app/app.component.spec.ts:1]: Oh then its fine.

**Review by JoyKiranSR — APPROVED** [review]
Code looks pretty clean for me. This is best to start with any angular project. Even eslint and codeql actions are also in place.
Great work out there!

**Comment by JoyKiran** [comment]
Thank you so much for taking time to review.

**Pull request merged**